### PR TITLE
Success flow for puzzles

### PR DESCRIPTION
**Pull request by jaydhulia**
- [x] Close popup after X seconds, on a puzzle being solved
- [x] Navigate to next (or current) scene on last object being solved

### CODE REVIEW

File: app/routes/ui/admin_scene.py [hunk 1]
```diff
@@ -33,15 +33,17 @@ async def get(self, scene_id):
 
             await self.render(
                 "scene.html",
-                is_last_scene=False,
+                is_last_scene="false",
```
Comment (ahmedhamodi) [inline app/routes/ui/admin_scene.py:36]: Any reason for string over the actual boolean value?

File: app/routes/ui/admin_scene.py [hunk 2]
```diff
@@ -33,15 +33,17 @@ async def get(self, scene_id):
...
                 is_admin=True,
                 scene_id=scene_id_int,
                 scenario_name="Edit Scene",
+                scenario_friendly_name="friendly_name",
                 scene_dict=scene_dict,
                 asset_prefix_url=config.get("asset.prefix_url"),
                 navmesh_src=navmesh_src,
                 inspector_url=config.get("inspector_url"),
                 json=json,
+                cur_scene_idx=scene_id_int,
```
Comment (ahmedhamodi) [inline app/routes/ui/admin_scene.py:46]: Why is this field needed?

**Review by ahmedhamodi — COMMENTED** [review]
Overall lgtm, couple of comments.

It seems to me that the new `cur_scene_idx` field isn't needed since it never gets updated? So can just use `scene_id` field unless I'm missing something.

File: app/routes/ui/admin_scene.py (hunk 2 above)
Comment (jaydhulia) [reply app/routes/ui/admin_scene.py:46]: `scene_id` is not the same as `cur_scene_idx`. `scene_id` is the id of the scene in postgres, while `cur_scene_idx` is the index of the current scene.
Example:
`scenario: [5, 77, 32, 23]`, then if someone opens the scenario: `scene_id` would be 5, and `cur_scene_idx` would be 0.

File: app/routes/ui/admin_scene.py (hunk 1 above)
Comment (jaydhulia) [reply app/routes/ui/admin_scene.py:36]: Javascript vs Python boolean. `False` is an undefined variable in HTML/JS, whereas "false" works

File: app/routes/ui/admin_scene.py (hunk 2 above)
Comment (ahmedhamodi) [reply app/routes/ui/admin_scene.py:46]: On lines 38 and 46, the two fields `scene_id` and `cur_scene_idx` are set to the same thing, so is this a mistake then?

Comment (jaydhulia) [reply app/routes/ui/admin_scene.py:46]: Yeah, that wasn't me and was leftover from another PR (https://github.com/uwblueprint/cdc-backend/pull/78) by accident. I can remove it, but it's not a biggie.

Comment (jaydhulia) [reply app/routes/ui/admin_scene.py:45]: Removed the unused variable `scene_id`

Comment (ahmedhamodi) [reply app/routes/ui/admin_scene.py:45]: Ah thank you. Makes sense now!

**Review by ahmedhamodi — APPROVED** [review]

**Pull request merged**